test/jit/test_torchbind.py

```diff
@@ -130,6 +130,74 @@ def foo():
         self.assertEqual(out[1].pop(), "mom")
         self.assertEqual(out[1].pop(), "hi")
 
+    def test_torchbind_def_property_getter_setter(self):
+        def foo_getter_setter_full():
+            fooGetterSetter = torch.classes._TorchScriptTesting._FooGetterSetter(5, 6)
+            # getX method intentionally adds 2 to x
+            old = fooGetterSetter.x
+            # setX method intentionally adds 2 to x
+            fooGetterSetter.x = old + 4
+            new = fooGetterSetter.x
+            return old, new
+        self.checkScript(foo_getter_setter_full, ())
+
+        def foo_getter_setter_script_check_outside():
+            fooGetterSetter = torch.classes._TorchScriptTesting._FooGetterSetter(5, 6)
+            return fooGetterSetter
+        scripted_foo_getter_setter = torch.jit.script(foo_getter_setter_script_check_outside)
+        value = scripted_foo_getter_setter()
+        self.assertEqual(value.x, 7)
+        value.x = 9
+        self.assertEqual(value.x, 13)
```

Comment:
Doesn't `checkScript` test this? It tests Python + both script frontends?

Reply:
ohh that's true. Will update

Reply:
just removed it because the test above already tests this behavior